canonicalwebteam/search/views.py

```diff
@@ -46,18 +44,16 @@ def build_search_view(
         )
     """
 
+    app = flask.Flask(__name__)
+    limiter = Limiter(
+        app, key_func=get_remote_address, default_limits=[request_limit]
```

Comment:
Out of scope for this PR, but @samhotep we need IP address for search to track people who spam us and store the rate limit in memory, since IPs are proxied, what would you recommend for this? I guess we can't use the same solution of the telephone number because this endpoint could be any `?search=xxx` and this module only accepts a remote address function.

Reply:
We'd need to force the `content-cache` to forward a custom header with the ip address probably, to preserve both the `content-cache` and `remote` IPs, and then update the limiter to use that header..
It's just an idea for now, but I'll check to see if the `content-cache` supports this